index.test.js

```diff
@@ -0,0 +1,122 @@
+import request from "supertest";
+import app from "./index.js";
+
+/*
+ *   This test  needs to run fully
+ *   it creates a user to test, without the
+ *   userTest all the non-POST request will fail
+ */
+
+describe("GET to /persons", () => {
+    test("/persons returns something", async () => {
+        const response = await request(app).get("/persons")
+        expect(response.text).toEqual(expect.stringContaining(""))
```

Comment:
When no person is found, you shouldn't return an empty array ("[]"), not an empty string :)

Reply:
I couldn't figure out a way to test the second case(where this path returns nothing 404), so I made this test only to see if the path returns anything, empty or a list.
I'm figuring out how to do it, so its just a placeholder for now(I should added a comment with it)

Reply:
Sorry. I wrote it wrong... "you should return an empty array"...

Reply:
In case nothing was found